docs/ossm/ossm2-migration/migration-checker.sh

```diff
@@ -26,8 +26,11 @@ BLANK='\033[0m'
 WARNING_EMOJI='\u2757'
 SPACER="-----------------------"
 
+# TODO: autodetect latest versions
 LATEST_VERSION=v2.6
 LATEST_CHART_VERSION=2.6.4
+LATEST_KIALI_VERSION=2.4.0
```

Comment:
For kiali cr and spec.version it should be `v2.4` and for some reason, currently the latest kiali operator is already `2.4.2`:
Your Kiali is not on the latest version. Current version: 'v2.4'. Latest version: '2.4.0'. Please upgrade your Kiali to the latest version.
Your Kiali operator is not on the latest version. Current version: '2.4.2'. Latest version: '2.4.0'. Please upgrade your Kiali operator to the latest version.